src/mame/toaplan/toaplan_v25_tables.h

```diff
@@ -109,15 +109,15 @@ class toaplan_v25_tables
 	// some kind of branch, not sure which
 	// it's used after compares in blocks, sometimes with a 'be' then a 'br' straight after, so it must be a condition that could also fail a be and fall to the br
 	//#define G_B0  0x74
-	#define G_B0  0x79
+	#define G_B0  0x79 // very low confidence
```

Comment:
The context suggests that this might instead be 0x77 (it follows comparing a bit position against 7).